README.md

```diff
@@ -113,6 +113,36 @@ CMake helpers. These will be automatically added to your
   * `cpplint` and `cppcheck` configuration
   * Logging configuration
 
+### Internationalization (i18n)
+
+Leatherman and its components provide support for generating and using
+`gettext`-based message catalogs. Two helpers are provided for
+generating message catalogs:
+
+* `gettext_templates <dir> <glob>`: (re)generates the .pot file from
+  source files matching the glob pattern, and updates any translation
+  (.po) files that already exist. Files are put in `dir`. New languages
+  can be added by configuring cmake with LEATHERMAN\_LOCALES.
+* `gettext_compile <dir> <inst>`: creates a `translation` target (also
+  used by `all`) to generate the binary message catalogs (.mo files) and
+  configure installing them to the specified install location (`inst`).
+
+`LEATHERMAN\_LOCALES` expects a quoted semi-colon separated list, as
+in `LEATHERMAN\_LOCALES="en;fr;ja"`.
+
+Normal use of cmake/make should ensure the translation files are up-to-
+date. Translations can be tested by setting the LC\_CTYPE environment
+variable.
+
+To translate strings outside of logging, use the
+`leatherman::locale::translate` and `leatherman::locale::format`
+helpers. Strings passed to the helpers will be extracted to .po files.
+`leatherman::locale::format` is a drop-in replacement for
+`boost::format`.
```

Comment:
It's boost::locale::format, link to http://www.boost.org/doc/libs/1_56_0/libs/locale/doc/html/localized_text_formatting.html.